### PR TITLE
Added  Release Checklist for Homepage

### PR DESCRIPTION
**Pull request by apeltzer**
x-ref: https://github.com/nf-core/tools/issues/355

### CODE REVIEW

File: markdown/developers/adding_pipelines.md [hunk 1]
```diff
@@ -285,3 +285,18 @@ Sometimes, especially when adding new features to a pipeline, the dependencies c
 * Create a Pull Request including only these two changes against the `dev` branch of the pipeline you're working on
 
 This way, a review process will be very fast and we can merge the changes into the `dev` branch, updating the Docker Image for that pipeline automatically. After ~30 Minutes, the Docker Image for that pipeline is then updated, and you can open your Pull Request containing your actual pipeline code changes.
+
+### Release Checklist
+
+Whenever a pipeline is considered ready for a (first) release, developers can simply follow these steps to make sure that the pipeline meets the requirements of the nf-core community. The respective pipeline should follow the Guidelines described above in the Guidelines in general. Then developers can follow these steps to get the pipeline to release:
```
Comment (drpatelh) [inline markdown/developers/adding_pipelines.md]:
```suggestion
As described above, when your pipeline is ready for a (first) release please follow the steps below to ensure that the pipeline meets the requirements of the nf-core community:
```

File: markdown/developers/adding_pipelines.md [hunk 2]
```diff
@@ -285,3 +285,18 @@ Sometimes, especially when adding new features to a pipeline, the dependencies c
...
+
+* Bump the `dev` branch containing the changes for the release to a release version (e.g. 1.0.0): `nf-core bump-version 1.0.0`
```
Comment (drpatelh) [inline markdown/developers/adding_pipelines.md]:
```suggestion
* Bump the version number of the `dev` branch to a release version (e.g. 1.0.0dev > 1.0.0): `nf-core bump-version 1.0.0`
```

File: markdown/developers/adding_pipelines.md [hunk 3]
```diff
@@ -285,3 +285,18 @@ Sometimes, especially when adding new features to a pipeline, the dependencies c
...
+* Check for pipeline dependencies that are out of date and update these accordingly in the `dev` branch: `nf-core lint .` will tell you which ones are outdated via automated API calls to (bio-) conda
```
Comment (drpatelh) [inline markdown/developers/adding_pipelines.md]:
```suggestion
* If applicable, update the software dependencies for the pipeline on the `dev` branch: `nf-core lint .` will tell you which ones are outdated via automated API calls to (bio-) conda
```

File: markdown/developers/adding_pipelines.md [hunk 4]
```diff
@@ -285,3 +285,18 @@ Sometimes, especially when adding new features to a pipeline, the dependencies c
...
+* Update the `CHANGELOG`, listing everything that has been added/fixed in this release
+* [Open a Pull Request (PR)](https://help.github.com/en/articles/creating-a-pull-request) from `dev` to `master` on GitHub after adjusting all of this - waiting for tests to pass
```
Comment (drpatelh) [inline markdown/developers/adding_pipelines.md]:
```suggestion
* [Open a Pull Request (PR)](https://help.github.com/en/articles/creating-a-pull-request) from `dev` to `master` on GitHub after adjusting all of this and wait for all of the tests to pass
```

File: markdown/developers/adding_pipelines.md [hunk 5]
```diff
@@ -285,3 +285,18 @@ Sometimes, especially when adding new features to a pipeline, the dependencies c
...
+* Link some reviewers (2 are required for merging to `master`) - asking for a final review on your release
```
Comment (drpatelh) [inline markdown/developers/adding_pipelines.md]:
```suggestion
* Request some reviewers (2 are required for merging to `master`) asking for a final review before your release
```

File: markdown/developers/adding_pipelines.md [hunk 6]
```diff
@@ -285,3 +285,18 @@ Sometimes, especially when adding new features to a pipeline, the dependencies c
...
+* Once approved by two reviewers, merge your PR into `master`
+* Go to GitHub and [create a new release for your pipeline](https://help.github.com/en/articles/creating-releases)
+  * Please make sure to use strictly numeric release numbers, most people follow [Semantic Versioning](https://semver.org/), e.g. 1.0.0, 1.0.
+* Optional: Use a [nice code name](http://www.codenamegenerator.com/) for your pipeline release
+* Create your release - tests will automatically run, DockerHub will generate a tagged container for that release.
```
Comment (drpatelh) [inline markdown/developers/adding_pipelines.md]:
```suggestion
* Create your release - tests will run automatically and DockerHub will generate a tagged container for that release.
```

Comment (drpatelh) [reply markdown/developers/adding_pipelines.md]: Looks good @apeltzer . Maybe need something about Zenodo IDs in the sections above and/or here i.e. need to make sure that the pipeline is linked on GitHub via WebHooks and then will need to create an additional PR to `master` via a branch to add the Zenodo ID by commenting out the code in the `Citation` section in the template.

**Comment by ewels** [comment]
You'll need to modify `header.php` to add a link to this new page from the navigation.. :)

**Comment by ewels** [comment]
Ignore me, this is adding to an existing page. Everything is fine. Move along..

File: markdown/developers/adding_pipelines.md (hunk 6 above)
Comment (apeltzer) [reply markdown/developers/adding_pipelines.md]: Very true! I'll add this now, too!

**Comment by apeltzer** [comment]
Added Zenodo too 👍

**Pull request merged**